src/Avalonia.Controls/Calendar/CalendarDatePicker.cs

```diff
@@ -425,9 +425,6 @@ static CalendarDatePicker()
         {
             FocusableProperty.OverrideDefaultValue<CalendarDatePicker>(true);
 
-            DisplayDateProperty.Changed.AddClassHandler<CalendarDatePicker>((x,e) => x.OnDisplayDateChanged(e));
-            DisplayDateStartProperty.Changed.AddClassHandler<CalendarDatePicker>((x,e) => x.OnDisplayDateStartChanged(e));
-            DisplayDateEndProperty.Changed.AddClassHandler<CalendarDatePicker>((x,e) => x.OnDisplayDateEndChanged(e));
```

Comment:
Do we need these private methods? If they aren't used, can you remove them?
OnDisplayDateChanged, OnDisplayDateStartChanged, OnDisplayDateEndChanged...

Reply:
🙏 Thank you for the feedback. Your feedback is always valuable to me.

Yes, I think I can remove these private methods. Also I can remove the private set methods. 
I just pushed the changes. Hopefully I didn't break anything else 😄